vnext/ReactWindowsCore/IWebSocketResource.h

```diff
@@ -20,62 +21,66 @@ struct IWebSocketResource {
 
 #pragma endregion Aliases
 
+// clang-format off
 #pragma region Inner types
```

Comment:
please remove all these, it just adds noise

Reply:
They are placed in specific spots for readability, especially in this interface.
This has been discussed offline, and the exceptions were deemed acceptable.

Reply:
As Alex raised in email, there are [other options](https://www.reddit.com/r/cpp_questions/comments/exrqtm/clangformat_align_enum_values/) to solve this problem that seem better than a blanket disable of formatting for the block.